libraries/Microsoft.Bot.Builder/TranscriptLoggerMiddleware.cs

```diff
@@ -104,15 +105,23 @@ public async Task OnTurnAsync(ITurnContext turnContext, NextDelegate nextTurn, C
             // flush transcript at end of turn
             while (transcript.Count > 0)
             {
-                try
-                {
-                    var activity = transcript.Dequeue();
-                    await logger.LogActivityAsync(activity).ConfigureAwait(false);
-                }
-                catch (Exception err)
-                {
-                    System.Diagnostics.Trace.TraceError($"Transcript logActivity failed with {err}");
-                }
+                var activity = transcript.Dequeue();
+
+#pragma warning disable 4014
```

Comment:
What are you disabling here? (per @daveta) (who can't type as quickly right now)